helm/oauth2-proxy/values.yaml

```diff
@@ -32,6 +32,15 @@ config:
   # If left empty, it will default to the release name
   cookieName: ""
   google: {}
+
```

Comment:
Please move the example config for google back here under the correct block. It is currently below your new `legacyDefaults` block

Reply:
Just fixed it. Please reopen if I missed something.